addons/mail/static/tests/discuss/core/web/messaging_menu.test.js

```diff
@@ -259,3 +266,33 @@ test("counter is updated on receiving message on non-fetched channels", async ()
     );
     await contains(".o-mail-MessagingMenu-counter", { text: "1" });
 });
+
+test("can use notification item swipe actions", async () => {
+    mockTouch(true);
+    patchUiSize({ size: SIZES.SM });
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({ name: "Demo", email: "demo@odoo.com" });
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "chat",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ partner_id: partnerId }),
+        ],
+    });
+    pyEnv["mail.message"].create({
+        author_id: partnerId,
+        body: "A message",
+        message_type: "comment",
+        model: "discuss.channel",
+        res_id: channelId,
+    });
+    await start();
+    await openDiscuss();
+    await contains("button.o-active", { text: "Notifications" });
+    await click("button", { text: "Chats" });
+    await contains(".o-mail-NotificationItem-badge:contains(1)");
+    await swipeRight(".o_actionswiper"); // marks as read
+    await contains(".o-mail-NotificationItem-badge", { count: 0 });
+    await swipeLeft(".o_actionswiper"); // unpins
+    await contains(".o-mail-NotificationItem", { count: 0 });
+});
```

Comment:
`await contains(".o-mail-NotificationItem", { count: 0 });` miss the positive assertion.
It's implicit with `.o-mail-Notificationitem-badge` and our BEM convention so that's alright, but just warning that without following BEM convention then this can be untrue. And also the root node needs to be added by hand so we could just miss adding it by hand over there.